src/olympia/versions/tasks.py

```diff
@@ -112,7 +113,36 @@ def delete_preview_files(pk, **kw):
 
 @task
 @use_primary_db
-def extract_version_to_git(version_id, author_id=None, note=None):
+def extract_addon_to_git(addon_pk):
```

Comment:
It's worth noting that once we solve the issue for the add-on code/repo, we can apply the same approach to source repo/code.